includes/dbFacile.php

```diff
@@ -114,9 +118,13 @@ function dbUpdate($data, $table, $where = null, $parameters = array()) {
 
 	// need field name and placeholder value
 	// but how merge these field placeholders with actual $parameters array for the WHERE clause
-	$sql = 'UPDATE `' . $table . '` set ';
+	$sql = 'UPDATE `' . $table . '` SET ';
 	foreach($data as $key => $value) {
-                $sql .= "`".$key."` ". '=:' . $key . ',';
```

Comment:
The NOW() value is called by a few functions like at the end of device discovery and poller.

 $sql .= "`".$key."` ". '=:' . $key . ',';
This can be changed back.  I see what it is doing in the dbMakeQuery.  Just looked weird at first.